restructuredtext_lint/test/test.py

```diff
@@ -71,10 +72,9 @@ def test_second_heading_short_line_number(self):
 
         This is a regression test for https://github.com/twolfson/restructuredtext-lint/issues/5
         """
-        filepath = os.path.join(__dir__, 'test_files', 'second_short_heading.rst')
```

Comment:
Upon re-reading this test, we should probably keep the filepath in here. It's an explicit regression test for short headings so calling it `warning_rst` loses context. I don't mind the loading of `warning_rst` at the top for the bottom tests -- mostly this usage of it